src/webparts/projectFilesViewer/components/ProjectFilesViewer.tsx

```diff
@@ -47,25 +48,36 @@ export default class ProjectFilesViewer extends React.Component<IProjectFilesVie
         documentPlace: "Documentos compartidos"
       }
     }
-    let id:string = document.URL.split('?')[1].split('&')[0].split('=')[1];
-    let site:string = document.URL.toLowerCase().split('project')[0];
-    let req:any = new XMLHttpRequest();
-    req.open('GET', site + `_api/ProjectData/${iProps.endpointPlace}?$format=json&$filter=${iProps.projectId}%20eq%20guid%27${id}%27`);
-    req.send('');
-    req.onreadystatechange = () =>{
-      if(req.readyState === 4){
-        let parsed = JSON.parse(req.response);
-        let projSite = iProps.urlProperty == "ProjectWorkspaceInternalUrl" ? parsed.value[0].ProjectWorkspaceInternalUrl : parsed.value[0].URLInternaDelEspacioDeTrabajoDelProyecto
-        this.setState({
-          url: projSite +`/${iProps.documentPlace}/Forms/AllItems.aspx`
-        })
+    let currentSiteUrl:any = document.URL.split("/");
+    (currentSiteUrl.indexOf("Project%20Detail%20Pages") == -1)? this.setState({isPDP:false}):this.setState({isPDP:true});
```

Comment:
Mutate the value of the isPDP state to a variable to use on the conditional `if`.

Reply:
done